src/main/java/byteceps/processing/HelpMenuManager.java

```diff
@@ -78,7 +78,7 @@ private String generateAllActions(String command) throws Exceptions.InvalidInput
         for (String flagFunction : flagFunctions) {
             result.append(String.format(HelpStrings.HELP_LIST_ITEM, flagFunction, System.lineSeparator()));
         }
-        result.delete(0, 4); //remove the first 3 tabs
+        result.delete(0, 13); //remove the first 12 whitespace characters
```

Comment:
Why remove the first 12 whitespace characters? Is it possible to remove the whitespace from HELP_LIST_ITEM or use `\t` to tab it?

Reply:
I replaced 3 tabs with 12 whitespace characters. I think _"\t"_ usage was causing the indentation issue because I believe the number of whitespace chars within tab varies depending on whether you use CMD or terminal, etc. So using a fixed number of whitespace characters is more consistent

Reply:
Also, I believe another teammate implemented result.delete(...) as the first bullet point is in line with [BYTE-CEPS] so it requires no such indentation